### PR TITLE
[P1-40] Erlocate error messages

### PR DESCRIPTION
**Pull request by CaptainExtremis**

### CODE REVIEW

**Review by Devanshusp — CHANGES_REQUESTED** [review]
Hey @CaptainExtremis, thanks for working on this!
For some reason, I am not able to type in the zip code field anymore-maybe something to do with the onChange event in the zip code input field.

Also, just for future reference, could you please add some screenshots of the changes you made? Helps a ton when checking prs!

File: src/components/VoterRegistration/pages/Eligibility/Eligibility.component.js [hunk 1]
```diff
@@ -146,6 +154,13 @@ export const Eligibility = () => {
             ...voterRegistrationData,
             dob: event.target.value,
           });
+          //set error code field to appear below dob field
+          if (voterRegistrationData.dob.length === 0) {
+            const dobInput = document.getElementById("dob");
+            dobInput.classList.add("requiredField");
+          }
+          setErrorCode(1);
+            return;
```
Comment (Devanshusp) [inline src/components/VoterRegistration/pages/Eligibility/Eligibility.component.js:163]: After removing this return, the zip code input works as expected.

**Review by Devanshusp — CHANGES_REQUESTED** [review]

**Comment by CaptainExtremis** [comment]
I've tried updating Instagram.js and Eligibility.component.js. See if you can merge them now.

**Comment by Devanshusp** [comment]
Hmm, still no luck; let me see what I can do

**Pull request closed without merging**